packages/my-queries-storage/src/pipeline-storage-schema.ts

```diff
@@ -62,8 +62,8 @@ export const PipelineSchema = z.preprocess(
     pipelineText: z.string(),
     lastModified: z
       .number()
-      .transform((x) => new Date(x))
-      .optional(),
+      .default(0)
```

Comment:
In the rarest case ever if someone still somehow doesn't have this value stored, we fallback to zero, the logic for writing this value to UserData was here for such a long time, I don't think this will ever be used